packages/website/src/App.tsx

```diff
@@ -122,6 +122,15 @@ function App(): React.ReactElement {
                       </>
                     )}
                   />
+                  <Route
+                    path="g/:customURL"
```

Comment:
I think this is going to be `/games/play/:customURL` instead of `/play/:customURL`

Reply:
should this route be higher?